tutorials/measurement_example.ipynb

```diff
@@ -146,7 +146,7 @@
     "## Defining a measurement chain: current measurement\n",
     "Now we define the missing elements of our measurement chain and bundle everything together. A core concept of the chain are `signals` that go in and out of `transformations` which define mathematical operations of the signals, forming the chain.\n",
     "\n",
-    "Each measurement chain starts with a `Source` signal. This is the point where our physical process or quantity is initially detected by a sensor. We define a `measurement.Source` object by giving a `name`, defining the `Signal` type and an `error` representing the uncertainty attached to the signal source. In a way, a `Source` is a special type of `transformation` in that it only has an `output_signal` but no `input_signal`.\n",
+    "Each measurement chain starts with a source signal. This is the point where our physical process or quantity is initially detected by a sensor. Since every `MeasurementChain` needs at least a source, we need to provide the corresponding data when creating an instance of this class. The parameters are the `source_name`, the `output_signal_type` and `output_signal_unit` as well as the `source_error` representing the uncertainty attached to the signal source. \n",
```

Comment:
```suggestion
    "Each measurement chain starts with a source signal. This is the point where our physical process or quantity is initially detected by a sensor. Since every `MeasurementChain` needs at least a source, we need to provide the corresponding information when creating an instance of this class. The parameters are the `source_name`, the `output_signal_type` and `output_signal_unit` as well as the `source_error` representing the uncertainty attached to the signal source. \n",
```
or maybe `metadata` here